covid_xprize/examples/prescriptors/neat/train_prescriptor.py

```diff
@@ -4,21 +4,26 @@
 # Example script for training neat-based prescriptors
 # Uses neat-python: pip install neat-python
 #
-
+import os
 from copy import deepcopy
 
 import neat
 import numpy as np
 import pandas as pd
-
-
+from pathlib import Path
 
 from covid_xprize.examples.prescriptors.neat.utils import PRED_CASES_COL, prepare_historical_df, CASES_COL, IP_COLS, \
     IP_MAX_VALUES, add_geo_id, get_predictions
 
 # Cutoff date for training data
 from covid_xprize.validation.cost_generator import generate_costs
 
+# Path where this script lives
+ROOT_DIR = Path(os.path.dirname(os.path.abspath(__file__)))
+
+# Config file for running NEAT (expected to reside in same dir as this script)
+NEAT_CONFIG_FILE = ROOT_DIR / 'config-prescriptor'
```

Comment:
Same change as above -- locate config file relative to where this script resides (same dir).